### ISSUE TITLE
Setting linear dimension "DIMSTYLE" via "Dimension" dialog

### ISSUE THREAD

**Issue by u2fly** [LibreCAD/LibreCAD#1867]
### Expected behavior

LibreCAD should support all standard DXF dimensions' `DIMSTYLE` variables.

And there should be an option to set/change default `DIMSTYLE` variables (e.g., `DIMTAD`, `DIMJUST`, etc.) manually for selected dimensions via _"Tools > Modify > Properties > Dimension"_ dialog.

- https://ezdxf.readthedocs.io/en/stable/dxfinternals/tables/dimstyle_table.html#text-location
- https://ezdxf.readthedocs.io/en/stable/tables/dimstyle_table_entry.html
- https://ezdxf.mozman.at/docs/tutorials/linear_dimension.html
- https://www.codeproject.com/articles/3398/cadlib-for-creating-dxf-drawing-interchange-format
- https://help.autodesk.com/view/ACD/2023/ENU/?guid=GUID-E063AF30-6F14-42D6-B0AB-0CF5C47B5897

AutoCAD's "DIMSTYLE: Modify" dialog:
![DIMSTYLE: Modify](https://help.autodesk.com/cloudhelp/2023/ENU/AutoCAD-Core/images/GUID-B8D34F09-C141-49B6-90CA-3A1A20E11169.png)

### Observed behavior

LibreCAD has no option to manually change default `DIMTAD` and `DIMJUST` for selected dimensions (via "Properties" or "Attributes"), but automatically switches some of this variables depending on dimensioned length and label text size.

### Steps to reproduce or sample file

1. Create drawing and add linear dimension;
2. Select dimension(s) and open _"Dimension"_ dialog (by double click or via _"Tools > Modify > Properties"_);
3. Try change `DIMSTYLE` variables (`DIMTAD`, `DIMJUST`, `DIMTOFL`, `DIMATFIT`, etc.) for **selected dimension**, to invert arrows, change label position, label text size, hide inner dimension line, etc.

![Screenshot_2024-08-31_00-44-17](https://github.com/user-attachments/assets/d62e7ef8-3bf2-4419-9e09-5ba641acba4c)

### Operating System and LibreCAD version info

All versions and platforms.

### Related issues

- https://github.com/LibreCAD/LibreCAD/issues/1864
- https://github.com/LibreCAD/LibreCAD/issues/1868

**Comment by sand1024** [comment]
yep, that's a good point. For sure, dimensions support should be extended, and, for example, support of ordinate dimensions in addition to linear also needed. 

However, support of all dimension types is not a quick feature to implement so it may require some time. 

Anyway, thank you for noting this issue - and it's great that you've also added references there, it will be simpler to implement support. 

I think we definitely need to support additional dimensions types.

**Comment by app4soft** [comment]
> For sure, dimensions support should be extended, and, for example, support of ordinate dimensions in addition to linear also needed.

I just updated issue description and added AutoCAD's "DIMENSION: Modify" dialog screenshot.

LibreCAD already has some of `DIMSTYLE` variables in "Current Drawing Prefernces > Dimensions" (applied to all dimensions in drawing), so adding missing options should be not too hard (as LibreCAD can render some `DIMENSIONS` features, where there is no option to set it manually, for example, invert dimension arrows direction).

But there is very limited set of options in "Dimensions" dialog (if use "Prefernces" on selected dimensions), so customizing `DIMSTYLE` per each dimension in LibreCAD is almost impossible as for now. I suppose there should be more options (already presented in "Current Drawing Prefernces > Dimensions") added into "Dimensions" dialog for selected dimension customizing.
